app/dashboard/browse/search/organizations/[id]/page.tsx

```diff
@@ -1,9 +1,113 @@
+"use client"
 import PageHeading from "@/components/dashboard/PageHeading";
+import { useToast } from "@/components/ui/use-toast";
+import { getOrgProfiles, getSingleOrg } from "@/utils/scripts/search";
+import { useEffect, useState } from "react";
+import Link from "next/link";
+
+export default function OrganizationPage({ params }: { params: { id: number } }) {
+
+  const { toast } = useToast();
+
+  const [errorFetching, setErrorFetching] = useState(false);
+  const [email, setEmail] = useState('');
+  const [bio, setBio] = useState('');
+  const [orgName, setOrgName] = useState('');
+  const [website, setWebsite] = useState('');
+  const [orgType, setOrgType] = useState(false);
+  const [orgProfiles, setOrgProfiles] = useState([]);
+
+
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const orgData = await getSingleOrg(params.id);
+      if (orgData === null) {
+        toast({
+          variant: "destructive",
+          title: "Unable to retrieve org profile",
+          description: "Reload page or check another time",
+        });
+        setErrorFetching(true);
+        return;
+      }
+      setOrgName(orgData.name);
+      setEmail(orgData.email);
+      setBio(orgData.bio);
+      setWebsite(orgData.website);
+      setOrgType(orgData.isNonProfit);
+
+      const orgProfilesData = await getOrgProfiles(params.id);
+      if (orgProfilesData === null) {
+        toast({
+          variant: "destructive",
+          title: "Unable to retrieve team profiles",
+          description: "Reload page or check another time",
+        });
+        setErrorFetching(true);
+        return;
+      }
+      setOrgProfiles(orgProfilesData);
+
+    }
+
+    fetchData();
+  }, []);
+
+  function Card({ id, name, email }: { id: string, name: string, email: string }) {
+    return (
+      <Link
+        href={`/dashboard/browse/search/${id}`}
+        className="border-2 border-neutral-400 bg-[#FFFEFE] 2xl:w-72 w-64 py-3 2xl:px-5 px-4 2xl:mr-5 mr-4 2xl:mb-4 mb-3 rounded-lg group hover:text-darkmaroon hover:border-darkmaroon transition duration-500"
+      >
+        <div className="font-semibold truncate">{name}</div>
+        <div className="text-sm text-body truncate group-hover:text-darkmaroon transition duration-500">
+          {email}
+        </div>
+      </Link>
+    );
+  }
 
-export default function OrganizationPage({ params }: { params: { id: string } }) {
   return (
     <div className="">
       <PageHeading header="Search" description="Organization profile" />
+      {errorFetching ? (
+        <div className="mt-8 text-center">
+          There was an error retrieving organization profile. Reload or try again later.
+        </div>
+      ) : (
+        <div className="mt-8">
+          <div className="2xl:w-[50vw] w-[50vw] 2xl:h-[65vh] h-[55vh] 2xl:px-10 px-6 2xl:py-8 py-6 mb-4 bg-[url('/assets/backgrounds/section_background.svg')] bg-no-repeat bg-cover bg-bottom rounded-3xl shadow-inner shadow-xl">
```

Comment:
@nam-t24 Can you confirm what this looks like on your screen? I chose 65vh because it helps cover the majority of whitespace on 1440p 27", but it does run to the bottom of the screen on 1080p 24"

Reply:
It's on the ss i put on my other review comments. the height is fine for screens under 2xl just text are too big

Reply:
If you want it to be better at responsiveness use fixed rem instead of vh since scaling based on screen size can scale really bad really fast

Reply:
Used fixed width and height for optimum viewability at 1080p resolutions